### PR TITLE
Opsdroid

### PR DESCRIPTION
**Pull request by aayushbisen**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

I have added **opsdroid** to the available swag opportunities.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -4269,13 +4269,15 @@
           "version": "1.0.0",
```
Comment (vikaspotluri123) [inline package-lock.json]: This file shouldn't have been changed!

File: data.json [hunk 2]
```diff
@@ -259,5 +259,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
```
Comment (vikaspotluri123) [inline data.json]: This file is alphabetical, sorted by name. Opsdroid should be earlier in the file 🙂

File: data.json [hunk 3]
```diff
@@ -259,5 +259,14 @@
...
+    "name": "opsdroid",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "name": "Opsdroid",
```

File: data.json [hunk 4]
```diff
@@ -259,5 +259,14 @@
...
+    "difficulty": "hard",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "difficulty": "medium",
```

Will reflect other difficulties (i.e. Umbraco)

File: data.json [hunk 5]
```diff
@@ -259,5 +259,14 @@
...
+    "description": "Contribute to opsdroid and earn a sticker pack!",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Contribute to opsdroid and receive some stickers!",
```

File: data.json [hunk 6]
```diff
@@ -259,5 +259,14 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
+  },
+  {
+    "name": "Opsdroid",
+    "difficulty": "medium",
+    "description": "Contribute to opsdroid and receive some stickers!",
```
Comment (aslafy-z) [inline data.json]: This `opsdroid` could also be uppercased

**Comment by aayushbisen** [comment]
Done.

File: data.json [hunk 7]
```diff
@@ -197,6 +197,15 @@
     "dateAdded": "2018-10-28T16:55:45.000Z",
     "tags": ["clothing", "hacktoberfest", "stickers"]
   },
+  {
+    "name": "Opsdroid",
+    "difficulty": "medium",
+    "description": "Contribute to opsdroid and receive some stickers!",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Contribute to Opsdroid and receive some stickers!",
```

File: data.json [hunk 8]
```diff
@@ -259,5 +268,5 @@
     "image": "https://developers.whatismybrowser.com/static/main/content/api/swag/whatismybrowser-stickers.jpg",
     "dateAdded": "2018-10-12T13:05:05.000Z",
     "tags": ["stickers"]
-  }
+  }
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
  }
```

**Comment by aayushbisen** [comment]
Hey @aslafy-z anything else to update in this PR?

**Comment by aslafy-z** [comment]
I changed back `opsdroid` to lowercase because I cannot find any mention of it with a first uppercase, see:

https://opsdroid.github.io/
https://medium.com/opsdroid
https://twitter.com/opsdroid

**Comment by aayushbisen** [comment]
Yes that's why I kept it lowercase before.

**Comment by aslafy-z** [comment]
Thanks for your contribution :fire: It's now live at [devSwag.io](https://devswag.io/) !

**Pull request merged**